### PR TITLE
Jaeger readme updates

### PR DESCRIPTION
**Pull request by adithyaravikumar**
Added documentation about getting started with the Jaeger-Swift project and having a simple setup. Also added a link to our demo app and some brief documentation about Opentracing.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,173 @@
...
+
+#### Mediating Jaeger Spans
+One of the challenges we ran into while building this project is that the Jaeger collector we report to only accepts data encoded in `Thrift`. We made an engineering decision to not perform this encoding on the client as it required an extra dependency. Therefore, we built a mediator service that takes in spans from the client, encodes this data to `Thrift Binary` format, and reports it to a sender.
+
+## Fitting everything together
+
+![jaeger sequence](https://user-images.githubusercontent.com/2333426/49960015-1d716b80-fedd-11e8-98b8-48727f34414d.png)
+
+When we create an instance of the Jaeger client, we specify two time intervals, a queuing time interval and a reporting time interval. When the app starts a new span, the span stores the timestamp at the exact moment. Once the span is finished, the span is queued in memory for the duration of the queuing time interval. Once this timer is invalidates, the agent picks up the spans and persists that data to disk. Now the second reporting timer picks up the spans from disk, reports the spans through the sender and then flushes the cache.
```
Comment (simonpierreroy) [inline README.md]: Once this timer is invalidates -> Once this timer is invalidated ?

File: README.md [hunk 2]
```diff
@@ -1,8 +1,173 @@
...
+```
+
+Now, if you open your browser and type [http://localhost:16686](http://localhost:16686), you should see the Jaeger web interface that displays trace information. Of course, there won't be any traces from our client there.
+
+###### Step 2 - Setup a mediator service
+The Jaeger collector only accepts data that is encoded in `Thrift` format. We have a mediator service that the client can report to. This mediator service accepts JSON spans from the Swift client, encodes this data to `Thrift Binary` and reports it to the Jaeger collector over UDP.
+
+* Download [our Jaeger mediator service](https://github.com/wayfair/jaeger-swift/tree/develop/Example/jaegerMediator).
+* By default, this mediator service reports spans to the locally hosted collector. If you already have a remote collector setup, you can specify the location in `configuration.json`.
+* Open Terminal. Run `npm ci` and then run `nom start`. You should now have the mediator service running on `http://localhost:3000`. You should see this message printed in the console
```
Comment (simonpierreroy) [inline README.md]: nom start -> npm start?

Comment (simonpierreroy) [inline README.md]: message printed in the console ->message printed in the console.

**Review by simonpierreroy — APPROVED** [review]

**Comment by simonpierreroy** [comment]
Nice work!!!!!!

File: README.md [hunk 3]
```diff
@@ -1,8 +1,173 @@
-# Jaeger for iOS
-
 [![Build Status](https://travis-ci.org/wayfair/jaeger-swift.svg?branch=master)](https://travis-ci.org/wayfair/jaeger-swift)
```
Comment (simonpierreroy) [inline README.md:1]: @aaronsky Why is this not working? 
screen shot 2018-12-13 at 5 09 02 pm

Comment (aaronsky) [reply README.md:1]: there are no CI builds for the master branch. This will work once we merge develop into master.

Comment (simonpierreroy) [reply README.md:1]: Thx!

File: README.md (hunk 2 above)
Comment (adithyaravikumar) [reply README.md]: Updated

File: README.md (hunk 3 above)
Comment (adithyaravikumar) [reply README.md:1]: It was pointing to master. Updated this to point to `develop` for now. Will need to update this when we eventually merge to master.

**Pull request merged**